nixos/modules/services/networking/prosody.nix

```diff
@@ -99,6 +99,33 @@ let
       description = "Push notifications to inform users of new messages or other pertinent information even when they have no XMPP clients online";
     };
 
+    cloud_notify_encrypted = mkOption {
```

Comment:
I only found out today that there is an extraModules option. We can just use that one, right?